### PR TITLE
docs(changelog): for v8.0.0 [skip ci]

### PR DESCRIPTION
**Pull request by tdeekens**
#### Summary

I gave this changelog my ❤️ and 👻. Hope it make sense. Feel free to comment or even push. Cheers.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,50 @@
...
+  - [#354](https://github.com/commercetools/merchant-center-application-kit/pull/354) fix(application-shell): redirect to welcome by default ([@tdeekens](https://github.com/tdeekens))
+
+Previously the `<ApplicationShell />` redirected to the `<projectKey>/dashboard` by default given the `/<projectKey>` route was matched exactly. This automatic redirect was removed. Instead our internal infrastructure (a proxy) now redirects to a welcome screen at `<projectKey>/welcome`. This means that your application can not rely anymore on being redirected to the dashboard.
+
+If you for instance had a:
+
+```jsx
+<Switch>
+  {process.env.NODE_ENV === 'production' ? null : (
+    <Redirect from="/:projectKey/dashboard" to="/:projectKey/welcome" />
```
Comment (emmenko) [inline CHANGELOG.md]: Maybe using a "diff" code snippet makes it simpler to show

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,50 @@
...
+## BREAKING CHANGES 💣
+
+This release introduces a **breaking change** which may entail some **migration steps**. We'll go through them now:
+
+#### 💥 Type: Breaking Change
+
+- `application-shell`
+  - [#354](https://github.com/commercetools/merchant-center-application-kit/pull/354) fix(application-shell): redirect to welcome by default ([@tdeekens](https://github.com/tdeekens))
+
+Previously the `<ApplicationShell />` redirected to the `<projectKey>/dashboard` by default given the `/<projectKey>` route was matched exactly. This automatic redirect was removed. Instead our internal infrastructure (a proxy) now redirects to a welcome screen at `<projectKey>/welcome`. This means that your application can not rely anymore on being redirected to the dashboard.
```
Comment (emmenko) [inline CHANGELOG.md]:
```suggestion
Previously the `<ApplicationShell>` had an internal redirect from `/:projectKey` to `/:projectKey/dashboard`. This redirect is now removed and it's done by our internal proxy router as we are rolling out a new landing page called **Welcome screen** (`/:projectKey/welcome`). However, for **development**, you need to adjust the redirect as illustrated below:
```

File: CHANGELOG.md (hunk 1 above)
Comment (emmenko) [reply CHANGELOG.md]:
```diff
<Redirect
+  exact={true}
-  from="/:projectKey/dashboard"
+  from="/:projectKey"
  to="/:projectKey/xxx"
/>
```

**Comment by emmenko** [comment]
I can do the changes myself later if you want.

**Comment by tdeekens** [comment]
> I can do the changes myself later if you want.

No worries. I am on it. Thanks!

**Comment by tdeekens** [comment]
Alright. I added the diff there. Thanks, better now. Anything else? @emmenko?

**Pull request merged**